src/main/java/uk/gov/dvsa/model/cvs/AdrPassCertificate.java

```diff
@@ -1,17 +1,16 @@
 package uk.gov.dvsa.model.cvs;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import uk.gov.dvsa.model.Document;
 import uk.gov.dvsa.model.cvs.certificateData.AdrPassCertificateData;
 import uk.gov.dvsa.model.cvs.certificateData.Signature;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
this seems to have been added for the ADR stuff. do we need to rebase 303030 from develop?